docs/en/deploy_old/environment_variables.md

```diff
@@ -0,0 +1,129 @@
+# Environment Variables
+
+### Setup environment variables
+
+Login to your application host
+
+dokku config:set loomio DATABASE_URL="postgresql://deploy:password@db.georgecloonio.local/loomio_georgecloo
+nio?sslmode=require"
+
+
+psql postgresql://deploy:everyoneattraction2@db.georgecloonio.local:5432/loomio_georgecloonio?sslmode=require
+```
+DATABASE_URL="postgresql://deploy:everyoneattraction2@db.georgecloonio.local:5432/loomio_georgecloonio?sslmode=require"
+dokku config:set loomio CANONICAL_HOST="www.georgecloonio.net"
+dokku config:set loomio DEFAULT_SUBDOMAIN="www"
+dokku config:set loomio MIN_THREADS=1
+dokku config:set loomio MAX_THREADS=1
+dokku config:set loomio PUMA_WORKERS=2
+dokku config:set loomio RAILS_ENV=production
+dokku config:set loomio SECRET_COOKIE_TOKEN=`< /dev/urandom tr -dc A-Za-z0-9 | head -c${1:-64};echo;`
+dokku config:set loomio DEVISE_SECRET=`< /dev/urandom tr -dc A-Za-z0-9 | head -c${1:-64};echo;`
+dokku config:set loomio TLD_LENGTH=1
+FORCE_SSL=1 # if you want ssl
+
+```
+ENV for plugins/optional things
+```
+ALLOW_ROBOTS:                1
+BING_TRANSLATE_APPID:        loomio
+BING_TRANSLATE_SECRET:      
+AWS_ACCESS_KEY_ID:
+AWS_SECRET_ACCESS_KEY:      
+AWS_UPLOADS_BUCKET:          loomio-uploads
+AIRBRAKE_API_KEY:           
+NEW_RELIC_APP_NAME:          loomio-production
+NEW_RELIC_ID:               
+NEW_RELIC_LICENSE_KEY:      
+NEW_RELIC_LOG:              
+TAG_MANAGER_ID:              
+INTERCOM_APP_API_KEY:       
+INTERCOM_APP_ID:            
+INTERCOM_APP_SECRET:        
+OMNI_CONTACTS_GOOGLE_KEY:   
+OMNI_CONTACTS_GOOGLE_SECRET:
+GOOGLE_KEY:                  
+GOOGLE_SECRET:               
+HEAP_APP_ID:                 
+FOG_DIRECTORY:               loomio-assets
+FOG_HOST:                    https://loomio-uploads.s3.amazonaws.com
+FOG_PROVIDER:                AWS
```

Comment:
Ah, remind me to sit and pair on this for setting up local file uploads, possibly ending up with a doc about it. Also possible we could debug why it's not working at the moment.